linode/manager
# tests: [M3-8656] - Mock LKE creation flow + APL coverage

## Description 📝
PR to remove any "real" entity creation during the create cluster flow and add APL coverage to the create specs.

> [!NOTE]
> The reason for mocking the main create flow is to avoid creating real clusters in the CI for security reasons.

## Changes  🔄
- Change main create flow to use mocks
- Support LKE creation with APL

## How to test 🧪

### Prerequisites

### Verification steps

- Run code locally + `yarn cy:debug` > select `lke-create`
- Confirm all other LKE suites are running fine in the CI

## As an Author, to speed up the review process, I considered 🤔

## As an Author, before moving this PR from Draft to Open, I confirmed ✅
